modules/odc-using-web-terminal.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// web_console/odc-about-web-terminal.adoc
+
+[id="odc-using-web-terminal_{context}"]
+= Using the web terminal
+
+After the *Web Terminal* Operator is installed, you can use the web terminal as follows:
+
+* Use the command line terminal icon on the upper right of the console to launch the web terminal. An, in context <confirm with sergei and abhi> web terminal, that uses your credentials, is displayed at the bottom of the screen.
```

Comment:
@abhinandan13jan @sleshchenko @gajanan-more can you confirm that the terminal instance has the context of the user, in terms of the project he is working on etc.?

Reply:
Is answered in https://github.com/openshift/openshift-docs/pull/28480#discussion_r554926693